CHANGES.rst

```diff
@@ -110,6 +110,13 @@ Unreleased
 - Started using ``exists`` query instead of ``count`` in ``populate_history`` command (gh-982)
 - Add basic support for many-to-many fields (gh-399)
 - Added support for Django 4.1 (gh-1021)
+- Added ``tracked_fields`` attribute to historical models (gh-1038)
+- Fixed ``KeyError`` when running ``clean_duplicate_history`` on models with ``excluded_fields`` (gh-1038)
+- Added support for Python 3.11 (gh-1053)
+- Fixed issue with deferred fields causing DoesNotExist error
+
+=======
+
```

Comment:
I think this is a rebasing conflict gone wrong?

The `Fixed issue with deferred fields causing DoesNotExist error` should go up into the unreleased section.

Reply:
Maybe, I'll move it there